Cargo.toml

```diff
@@ -28,6 +28,7 @@ members = ["proto", "benchmark", "compiler", "interop"]
 default = ["protobuf-codec"]
 protobuf-codec = ["protobuf"]
 secure = ["grpcio-sys/secure"]
+openssl = ["grpcio-sys/openssl"]
```

Comment:
How about add `secure` features as a dependency?

Reply:
Done. Also re-based vs master to fix build.